tests/integration_tests/test_simple_features.py

```diff
@@ -162,18 +162,18 @@ def test_feature(input_test_feature, output_test_feature,
 @pytest.mark.parametrize(
     'input_test_feature, output_test_feature',
     [
-        # ([category_feature()],
-        #  [binary_feature(), binary_feature()]),
-        # ([category_feature()],
-        #  [category_feature(vocab_size=5), category_feature(vocab_size=7)]),
-        # ([category_feature()],
-        #  [numerical_feature(), numerical_feature()]),
-        # ([category_feature()],
-        #  [sequence_feature(vocab_size=5), sequence_feature(vocab_size=7)]),
-        # ([category_feature()],
-        #  [set_feature(vocab_size=5), set_feature(vocab_size=7)]),
-        # ([category_feature()],
-        #  [text_feature(vocab_size=5), text_feature(vocab_size=7)]),
+        ([category_feature()],
+         [binary_feature(), binary_feature()]),
+        ([category_feature()],
+         [category_feature(vocab_size=5), category_feature(vocab_size=7)]),
+        ([category_feature()],
+         [numerical_feature(), numerical_feature()]),
+        ([category_feature()],
+         [sequence_feature(vocab_size=5), sequence_feature(vocab_size=7)]),
+        ([category_feature()],
+         [set_feature(vocab_size=5), set_feature(vocab_size=7)]),
+        ([category_feature()],
+         [text_feature(vocab_size=5), text_feature(vocab_size=7)]),
         ([category_feature()],
          [vector_feature(), vector_feature()]),
```

Comment:
this looks great for testing output vector features, although it doesn't seem to me we are testing for input vector features, or ar we?